README.rst

```diff
@@ -148,4 +181,79 @@ Run the example on REANA cloud
 We can now install the REANA client and submit the hello world example to run on
 some particular REANA cloud instance:
 
-**FIXME**
+
+First we provide the REANA Server instance URL and we test if it works using the `ping` command:
+
+.. code-block:: console
+
+   $ export REANA_SERVER_URL=http://192.168.99.100:31201
+   $ reana-client ping
+   [INFO] REANA Server URL ($REANA_SERVER_URL) is: http://192.168.99.100:31201
+   [INFO] Connecting to http://192.168.99.100:31201
+   [INFO] Server is running.
+
+Once this is done we can run the analysis:
```

Comment:
Looks a bit too long on the GitHub preview, perhaps we can break it into:

- Connect to the REANA cloud instance where we will run this example:
- Initialise workflow and upload input data and code:
- Start workflow execution and enquire about its running status:
- Download output files: